basic/src/main/resources/templates/home.html

```diff
@@ -0,0 +1,45 @@
+<!--
+  ~ Copyright 2017 Okta, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>User Details</title>
+    <!--/*/ <th:block th:include="head :: head"/> /*/-->
+</head>
+  <body id="samples">
+  <div th:replace="menu :: menu"></div>
+
+    <div id="content" class="container">
+      <h2>Okta Hosted Login + Spring Boot Example</h2>
+
+      <div th:unless="${#authorization.expression('isAuthenticated()')}">
+        <p>Hello!</p>
+        <p>If you're viewing this page then you have successfully configured this example server.</p>
+        <p>This example shows you how to use the <a href="https://github.com/okta/okta-spring-boot">Okta Spring Boot Starter</a> to add the <a href="https://developer.okta.com/authentication-guide/implementing-authentication/auth-code.html">Authorization Code Flow</a> to your application.</p>
+        <p>When you click the login button below, you will be redirected to the login page on your Okta org.  After you authenticate, you will be returned to this application.</p>
+      </div>
+
+      <div th:if="${#authorization.expression('isAuthenticated()')}">
+        <p>Welcome back, <span th:text="${#authentication.name}">Joe Coder</span>!</p>
+        <p>You have successfully authenticated against your Okta org, and have been redirected back to this application.</p>
+        <p>Visit the <a th:href="@{/profile}">My Profile</a> page in this application to view the information retrieved with your OAuth Access Token.</p>
```

Comment:
Just to confirm, is all the user profile info retrieved from OAuth Access Token or do we call the `/userinfo` endpoint?

Reply:
The access token claims are used by default, unless you set `okta.oauth2.localTokenValidation=false`

Reply:
We need to document about this & running tests locally. Probably another PR

Reply:
👍 It's not super relevant for this repo, but the README for the `okta-spring-boot` repo needs a once over